utils/generate_grammar_tables.py

```diff
@@ -53,6 +53,7 @@
 SPV_EXT_shader_viewport_index_layer
 SPV_AMD_shader_image_load_store_lod
 SPV_AMD_shader_fragment_mask
+SPV_GOOGLE_decorate_string
```

Comment:
I would include SPV_GOOGLE_hlsl_functionality1 here as well.
Both of these are equally represented in the core grammar file.

Reply:
I think `SPV_GOOGLE_hlsl_functionality1` is automatically picked up because it is listed as the extension for the `HlslSemanticGOOGLE` decoration. This one not becuase of #1368. But it does not harm to also list here.